js/content-script.js

```diff
@@ -162,7 +162,7 @@ function overrideRequireJs(b, extensionId) {
           if (p) {
             var jb = r(),
               kb = 0;
-            if (ib != null) kb = hb.stopTiming(ib);
+            if (ib !== null) kb = hb.stopTiming(ib);
```

Comment:
This change would make the code behave differently - `!= null` matches both `null` and `undefined`, whereas `!== null` only matches `null`.